model/clusters_mgmt/v1/cluster_type.model

```diff
@@ -96,6 +96,9 @@ class Cluster {
 	// Flag indicating if the cluster is BYOC (customer cloud subscription).
 	BYOC Boolean
 
+    	// Provisioner DNS Ready Status
+    	DNSReady Boolean
```

Comment:
This line seems to be using spaces for indentation, should be tabs

Reply:
I m not sure why my IDE doesn't show the indentation issue